js/script.js

```diff
@@ -0,0 +1,298 @@
+        list.appendChild(listItem);
+    }
+}
+
+// Create question block
+const createElemQuestion = () => {
+    const question = document.createElement('p');
+    question.classList.add('question_game');
+    hangmanKeyboards.appendChild(question);
+
+    return question;
+}
+const returnQuestion = createElemQuestion();
+
+// Create attempts block
+const createAttempts = () => {
+    const attempts = document.createElement('p');
+    attempts.classList.add('attempts_games');
+    attempts.innerHTML = `Incorrect guesses: <span>0 / 6</span>`; //Attempts block
+    hangmanKeyboards.appendChild(attempts);
+
+    return attempts;
+}
+const returnAttempt = createAttempts();
+
+// Create keyboard container
+const createKeyboardContainer = () => {
+    const keyboardsContainer = document.createElement('div');
+    keyboardsContainer.classList.add('keyboards__container'); //Keyboards Container
+    hangmanKeyboards.appendChild(keyboardsContainer);
+
+    return keyboardsContainer;
+}
+const keyboardContainer = createKeyboardContainer();
+
+// Create keyboard buttons
+const createKeyboard = (container) => {
+    for (let i = 65; i <= 90; i++) {
+        const keyboardsBtn = document.createElement('button');
+        keyboardsBtn.type = 'button';
+        keyboardsBtn.classList.add('keyboards__button');
+        keyboardsBtn.setAttribute('data-code', `Key${String.fromCharCode(i)}`);
+        keyboardsBtn.innerText = String.fromCharCode(i); // Keyboard
+        container.appendChild(keyboardsBtn);
+    }
+}
+createKeyboard(keyboardContainer);
+
+// KeyA => a
+const formaKeyboardCode = (code) => code[code.length - 1].toLowerCase();
+
+
+// Get Random questions
+const getRandomQuestion = (question) => {
+    const { word, hint } = wordList[Math.floor(Math.random() * wordList.length)];
+    console.log('correct answer: ', word);
+    question.innerHTML = 'Hint: ' + `<b>${hint}</b>`;
+    createListItem(listWords, word.length);
+
+    return word;
+}
+const answer = getRandomQuestion(returnQuestion);
+
+//Open modal window
+const callPopUpWindow = () => {
+    const ulList = document.querySelector('.hangmanGame__list');
+
+    return Array.from(ulList.children).every((item) => item.innerHTML);
+}
+
+const handleClick = (event) => {
+    let letter;
+    const ul = document.querySelector('.hangmanGame__list');
+    const liItems = Array.from(ul.children);
+    const button = document.querySelector(`[data-code = ${event.code}]`);
+    
+    // Disabled click buttons
+    const arrBtn = Array.from(keyboardContainer.children);
+
+    for(let i = 0; i <= arrBtn.length; i++) {
+        if(arrBtn[i] === event.target || arrBtn[i] === button) {
+            arrBtn[i].setAttribute('disabled', 'disabled');
+            arrBtn[i].style = 'background-color: rgb(151, 141, 158)';
+        }
+    }
+
+    if(event.code) {
+        letter = formaKeyboardCode(event.code);
+
+    } else {
+        letter = event.target.innerHTML;
```

Comment:
```suggestion
        letter = event.target.innerHTML.toLowerCase();
```
приводим к нижнему регистру и чиним клавиатуру